webapp/guides.py

```diff
@@ -22,13 +22,8 @@ def init_docs(app):
             index_topic_id=DISCOURSE_INDEX_TOPIC,
             url_prefix="/guides",
         ),
-        document_template="guides.html",
+        document_template="docs.html",
         url_prefix="/guides",
     )
 
-    @discourse_docs.blueprint.before_request
```

Comment:
I was not entirely sure we needed this one, I think we don't because we already do the same [here](https://github.com/canonical-web-and-design/webteam.canonical.com/pull/94/files#diff-8235c5f219f2fb4d725e22f674ae43d399f5464fa6e493af349e0d0641401bbcR54)